src/mode/modeVisual.ts

```diff
@@ -40,6 +40,9 @@ export class VisualMode extends Mode {
     }
 
     shouldBeActivated(key: string, currentMode: ModeName): boolean {
+        if (currentMode !== ModeName.Normal) {
+            return false;
+        }
         return key === "v";
```

Comment:
I think it's simpler if you do this:

`return key === "v" && currentMode === ModeName.Normal`

Can you also add in the `modeNormal.ts` that is you are in Visual Mode and `v` is entered, it should go to Normal Mode?